### ISSUE TITLE
Instead of adding  false music tags. Add option to not add any if not found the right tag

### ISSUE THREAD

**Issue by pakosaan** [deepjyoti30/ytmdl#242]
# Feature Request

## Checklist

- [ ] I've searched the bugtracker for similar feature requests including closed ones.
- [ ] I will add the enhancement label to the issue.

## Description

Describe the feature

**Comment by deepjyoti30** [comment]
@pakosaan Have you considered passing the option as `-1` when the prompt is shown to select the metadata? Entering `-1` will skip the selection and save the song without any meta.

**Comment by berinaniesh** [comment]
Well, its good to have metadata. Only after the search finishes, you search for 5 more options 5 times ( 25 tags total), you find that none of them match and then you will have to press some number or else the file is discarded (I guess). It would be nice to have an option after the tag options are presented to us, to skip writing tags.

**Comment by deepjyoti30** [comment]
Yes, that's how it's intended to work. After you look through all the tags, you can still press `-1` and writing meta will be skipped.

**Comment by deepjyoti30** [comment]
Since this feature is already added and is similar to #224 I am closing the issue.
